releases/latest/beta/Dockerfile.ubuntu.openjdk8

```diff
@@ -41,7 +17,7 @@ LABEL org.opencontainers.image.authors="Leo Christy Jesuraj, Melissa Lee, Thomas
       org.opencontainers.image.revision="$LIBERTY_BUILD_LABEL" \
       org.opencontainers.image.description="This image contains the Open Liberty beta runtime with IBM Semeru Runtime Open Edition OpenJDK 8 with OpenJ9 and Ubuntu as the base OS.  For more information on this image please see https://github.com/OpenLiberty/ci.docker#building-an-application-image" \
       org.opencontainers.image.title="Open Liberty Beta" \
-      org.opencontainers.image.version="${LIBERTY_VERSION}"
```

Comment:
Strange inconsistency.  Good to clean up.

Reply:
ah, I thought I caught all of these

Reply:
oh, I did.  This was the old version with the inconsistency

Reply:
Sorry I was unclear on my comment.  I was saying it is good you cleaned that up.